### PR TITLE
[theme/ricos-tiptap-editor] add editor-custom-style

### PR DESCRIPTION
**Pull request by tamrat-bay**

### CODE REVIEW

**Comment by oded-soffrin** [comment]
Notice how in groups the editor grows until it reaches its max height:

editorHeight

this is the css they apply:
image


is it possible with this API ?

**Comment by tamrat-bay** [comment]
> Notice how in groups the editor grows until it reaches its max height:
> 
> editorHeight [ editorHeight ](https://user-images.githubusercontent.com/1297297/190923903-20e322b7-c548-40ad-8cde-4f25d7554358.gif) [ ](https://user-images.githubusercontent.com/1297297/190923903-20e322b7-c548-40ad-8cde-4f25d7554358.gif)
> 
> this is the css they apply: image
> 
> is it possible with this API ?

It's possible, we just need to give them the option of
min-height and max-height in the theme

 // customStyles : {editor : { minHeight: '200px' ,maxHeight: 'calc(100vh - 460px)' } }

File: packages/ricos-types/web/src/themeTypes.ts [hunk 1]
```diff
@@ -75,6 +75,9 @@ export interface CustomMentionStyle {
 export interface CustomFooterToolbarStyle {
   marginTop?: CSSProperties['marginTop'];
 }
+export interface CustomEditorStyle {
+  height?: CSSProperties['height'];
```
Comment (tamrat-bay) [inline packages/ricos-types/web/src/themeTypes.ts:79]: add min & max options

File: packages/tiptap-editor/web/src/statics/styles/tiptap-editor-styles.scss [hunk 2]
```diff
@@ -1,4 +1,5 @@
 @import '~wix-rich-content-editor-common/dist/statics/styles/editor-styles';
+// @import '~wix-rich-content-common/dist/statics/styles/palette';
```
Comment (danarish4) [inline packages/tiptap-editor/web/src/statics/styles/tiptap-editor-styles.scss:2]: please remove 
```suggestion
```

**Review by danarish4 — CHANGES_REQUESTED** [review]
Looks good 🔥 
Please add this new API to our docs 
https://ricos.js.org/docs/ricos/theming/#custom-styles

How did you check it? You can add an editor height option to the theme storybook
https://rich-content-storybook-master.surge.sh/?path=/story/theme--custom-styles

**Comment by tamrat-bay** [comment]
> Looks good 🔥 Please add this new API to our docs [ricos.js.org/docs/ricos/theming/#custom-styles](https://ricos.js.org/docs/ricos/theming/#custom-styles)
> 
> How did you check it? You can add an editor height option to the theme storybook [rich-content-storybook-master.surge.sh/?path=/story/theme--custom-styles](https://rich-content-storybook-master.surge.sh/?path=/story/theme--custom-styles)

Currently, none of the theme are working in the storybook.
I passed the relevant property into the exampleApp and it looks great.

**Review by danarish4 — APPROVED** [review]
🔥

**Comment by maimonav** [comment]
@tamrat-bay 
> Currently, none of the theme are working in the storybook.
Not accurate, storybook include some stories with theme, for example [here](https://rich-content-storybook-master.surge.sh/?path=/story/theme--palette) , you can also toggle between draft & tiptap.

**Comment by oded-soffrin** [comment]
image

**Pull request merged**